types/react-native-modal-dropdown/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for react-native-modal-dropdown 0.6
+// Project: https://github.com/nitin42/react-native-modal-dropdown
```

Comment:
Typo? This URL is a 404. I see https://github.com/sohobloo/react-native-modal-dropdown on npm.